src/core/EntryAttachments.cpp

```diff
@@ -151,3 +152,17 @@ bool EntryAttachments::operator!=(const EntryAttachments& other) const
 {
     return m_attachments != other.m_attachments;
 }
+
+int EntryAttachments::attachmentsSize(const QSet<QByteArray> &ignoredAttachments) const
+{
+    int size = 0;
+
+    QMapIterator<QString, QByteArray> i(m_attachments);
+    while (i.hasNext()) {
+        i.next();
+        if( ! ignoredAttachments.contains( i.value() )){
```

Comment:
Add space after if and before {, remove spaces around !
Also no spaces around i.value()